pyrk/th_system.py

```diff
@@ -80,27 +247,100 @@ def convection(self, t_b, t_env, h, A):
         :type A: float.
         """
         num = (t_b-t_env)
-        denom = (1.0/(h*A))
+        denom = (1.0/(h.magnitude*A.magnitude))
         return num/denom
 
-    def conduction(self, t_b, t_env, L, k, A):
+    def custom(self, t_b, t_env, res):
+        num = (t_b-t_env)
+        denom = res.to(units.kelvin/units.watt)
+        return num/denom
+
+
+class THSystemSphPS(THSystem):
+
+    """This class models heat transfer in a sphere with point heat source at the
+    center and convective heat transfer between the sphere and liquid flow
+
+    !!NOT MAITEINED
```

Comment:
delete this class if it is not maintained. It shouldn't be in the code if it's not worth maintaining.